### PR TITLE
Add capistrano to ealapps

### PR DESCRIPTION
**Pull request by christinach**
closes https://github.com/pulibrary/princeton_ansible/issues/3482

part of https://github.com/PrincetonUniversityLibrary/ealapps/issues/1

### CODE REVIEW

File: roles/ealapps/tasks/main.yml [hunk 1]
```diff
@@ -25,10 +10,26 @@
...
 
+- name: ealapps | create directories for shared files
+  ansible.builtin.file:
+    path: '{{ capistrano_base_dir }}/{{ item }}'
+    state: 'directory'
+    owner: '{{ deploy_user }}'
+    group: '{{ deploy_user }}'
+    mode: 0755
+  changed_when: false
+  with_items:
```
Comment (kayiwa) [inline roles/ealapps/tasks/main.yml:21]: can we make this use the preferred [loops](https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_loops.html)

**Comment by acozine** [comment]
We can also remove https://github.com/pulibrary/princeton_ansible/tree/main/roles/ealapps/files now that deployment is happening with capistrano.

**Comment by maxkadel** [comment]
> We can also remove https://github.com/pulibrary/princeton_ansible/tree/main/roles/ealapps/files now that deployment is happening with capistrano.

Done!

File: roles/ealapps/tasks/main.yml [hunk 2]
```diff
@@ -25,10 +10,26 @@
...
+  ansible.builtin.file:
+    path: '{{ capistrano_base_dir }}/{{ item }}'
+    state: 'directory'
+    owner: '{{ deploy_user }}'
+    group: '{{ deploy_user }}'
+    mode: 0755
+  changed_when: false
+  loop:
+    - '{{ capistrano_directory }}/shared/EALJ'
+    - '{{ capistrano_directory }}/shared/EALJ/pdfs'
```
Comment (acozine) [inline roles/ealapps/tasks/main.yml:23]: This directory, or a sub-directory called /shared/EALJ/pdfs/tmp/ needs to have 0666 perms so individual articles can be served up to the user. Add a task for this.

**Comment by acozine** [comment]
Current CI failure is a linting issue:

```
Warning: [truthy] truthy value should be one of [false, true]
Warning: [truthy] truthy value should be one of [false, true]
Error: [empty-lines] too many blank lines (1 > 0)
```

**Comment by acozine** [comment]
The `ealapps` role is passing now, but the `ojs` tests are failing with this error:
```
TASK [ojs : ojs | run expect script for OJS interactive installation] **********
  fatal: [instance]: FAILED! => {"changed": true, "cmd": ["/usr/bin/expect", "/var/www/ojs-3.3.0-8/script.exp"], "delta": "0:00:00.211800", "end": "2022-12-13 19:52:20.372733", "msg": "non-zero return code", "rc": 1, "start": "2022-12-13 19:52:20.160933", "stderr": "send: spawn id exp3 not open\n    while executing\n\"exp_send -s -- $arg\"\n    (procedure \"send\" line 3)\n    invoked from within\n\"send -- \"en_US\\r\"\"\n    (file \"/var/www/ojs-3.3.0-8/script.exp\" line 49)", "stderr_lines": ["send: spawn id exp3 not open", "    while executing", "\"exp_send -s -- $arg\"", "    (procedure \"send\" line 3)", "    invoked from within", "\"send -- \"en_US\\r\"\"", "    (file \"/var/www/ojs-3.3.0-8/script.exp\" line 49)"], "stdout": "spawn php8.1 tools/install.php\r\nPHP Fatal error:  Uncaught Error: Call to undefined function xml_parser_create() in /var/www/ojs-3.3.0-8/lib/pkp/classes/xml/PKPXMLParser.inc.php:212\r\nStack trace:\r\n#0 /var/www/ojs-3.3.0-8/lib/pkp/classes/xml/PKPXMLParser.inc.php(162): PKPXMLParser->createParser()\r\n#1 /var/www/ojs-3.3.0-8/lib/pkp/classes/xml/PKPXMLParser.inc.php(204): PKPXMLParser->parseTextStruct()\r\n#2 /var/www/ojs-3.3.0-8/lib/pkp/classes/db/XMLDAO.inc.php(52): PKPXMLParser->parseStruct()\r\n#3 /var/www/ojs-3.3.0-8/lib/pkp/classes/i18n/TimeZoneDAO.inc.php(58): XMLDAO->parseStruct()\r\n#4 /var/www/ojs-3.3.0-8/lib/pkp/classes/cache/GenericCache.inc.php(63): TimeZoneDAO->_timeZoneCacheMiss()\r\n#5 /var/www/ojs-3.3.0-8/lib/pkp/classes/cache/FileCache.inc.php(115): GenericCache->get()\r\n#6 /var/www/ojs-3.3.0-8/lib/pkp/classes/i18n/TimeZoneDAO.inc.php(77): FileCache->getContents()\r\n#7 /var/www/ojs-3.3.0-8/lib/pkp/classes/i18n/PKPLocale.inc.php(885): TimeZoneDAO->getTimeZones()\r\n#8 /var/www/ojs-3.3.0-8/lib/pkp/classes/i18n/PKPLocale.inc.php(300): PKPLocale::getTimeZone()\r\n#9 /var/www/ojs-3.3.0-8/lib/pkp/classes/cliTool/CliTool.inc.php(64): PKPLocale::initialize()\r\n#10 /var/www/ojs-3.3.0-8/tools/install.php(26): CommandLineTool->__construct()\r\n#11 /var/www/ojs-3.3.0-8/tools/install.php(47): OJSInstallTool->__construct()\r\n#12 {main}\r\n  thrown in /var/www/ojs-3.3.0-8/lib/pkp/classes/xml/PKPXMLParser.inc.php on line 212", "stdout_lines": ["spawn php8.1 tools/install.php", "PHP Fatal error:  Uncaught Error: Call to undefined function xml_parser_create() in /var/www/ojs-3.3.0-8/lib/pkp/classes/xml/PKPXMLParser.inc.php:212", "Stack trace:", "#0 /var/www/ojs-3.3.0-8/lib/pkp/classes/xml/PKPXMLParser.inc.php(162): PKPXMLParser->createParser()", "#1 /var/www/ojs-3.3.0-8/lib/pkp/classes/xml/PKPXMLParser.inc.php(204): PKPXMLParser->parseTextStruct()", "#2 /var/www/ojs-3.3.0-8/lib/pkp/classes/db/XMLDAO.inc.php(52): PKPXMLParser->parseStruct()", "#3 /var/www/ojs-3.3.0-8/lib/pkp/classes/i18n/TimeZoneDAO.inc.php(58): XMLDAO->parseStruct()", "#4 /var/www/ojs-3.3.0-8/lib/pkp/classes/cache/GenericCache.inc.php(63): TimeZoneDAO->_timeZoneCacheMiss()", "#5 /var/www/ojs-3.3.0-8/lib/pkp/classes/cache/FileCache.inc.php(115): GenericCache->get()", "#6 /var/www/ojs-3.3.0-8/lib/pkp/classes/i18n/TimeZoneDAO.inc.php(77): FileCache->getContents()", "#7 /var/www/ojs-3.3.0-8/lib/pkp/classes/i18n/PKPLocale.inc.php(885): TimeZoneDAO->getTimeZones()", "#8 /var/www/ojs-3.3.0-8/lib/pkp/classes/i18n/PKPLocale.inc.php(300): PKPLocale::getTimeZone()", "#9 /var/www/ojs-3.3.0-8/lib/pkp/classes/cliTool/CliTool.inc.php(64): PKPLocale::initialize()", "#10 /var/www/ojs-3.3.0-8/tools/install.php(26): CommandLineTool->__construct()", "#11 /var/www/ojs-3.3.0-8/tools/install.php(47): OJSInstallTool->__construct()", "#12 {main}", "  thrown in /var/www/ojs-3.3.0-8/lib/pkp/classes/xml/PKPXMLParser.inc.php on line 212"]}
```

**Comment by sandbergja** [comment]
I rebased this branch, with the goal of getting b09d44a9c9ee9850e940368273bd86d6779d9456 into this branch so the ojs tests would pass.

**Review by sandbergja — APPROVED** [review]
Thanks, @christinach , @tventimi , @maxkadel , @acozine , @rladdusaw , @leefaisonr , @VickieKarasic , @bess , @carolyncole , and @kayiwa !  Truly a group effort.

**Pull request merged**